semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/LiteralMethods.scala

```diff
@@ -1,16 +1,43 @@
 package io.shiftleft.semanticcpg.language.nodemethods
 
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.Languages
 import io.shiftleft.codepropertygraph.generated.nodes.{Literal, NewLocation}
 import io.shiftleft.semanticcpg.NodeExtension
 import io.shiftleft.semanticcpg.language.*
 
 class LiteralMethods(val literal: Literal) extends AnyVal with NodeExtension with HasLocation {
-  def value(typeFullName: String): String = {
-    if (literal.typeFullName == typeFullName) {
-      return "asd"
+  def innerText: Option[String] = {
+    val delimiters = Cpg(literal.graph).metaData.language.headOption match {
```

Comment:
This mapping should be defined in a static field outside, maybe in the companion object.

Reply:
@max-leuthaeuser moved this into a `Map` in `object LiteralMethods`. Let me know if this is what you wanted or if I should change it to something else.